plugins/node/plugin.zsh

```diff
@@ -0,0 +1,21 @@
+# Color definitions
+GEOMETRY_COLOR_NODE_NPM_VERSION=${GEOMETRY_COLOR_NODE_NPM_VERSION:-black}
+
+# Symbol definitions
+GEOMETRY_SYMBOL_NODE_NPM_VERSION=${GEOMETRY_SYMBOL_NODE_NPM_VERSION:-"⬡"}
+GEOMETRY_NODE_NPM_VERSION=$(prompt_geometry_colorize $GEOMETRY_COLOR_NODE_NPM_VERSION $GEOMETRY_SYMBOL_NODE_NPM_VERSION) 
+
+
+geometry_prompt_node_setup(){} 
+
+geometry_prompt_node_render(){	
+    if (( $+commands[node] == 0 )); then
```

Comment:
Remember that `$+commands[node]` will return `0` if the command does not exists, `1` otherwise. So, this conditional is backwards.